.gitlab/validate-agent-build/trigger_agent_build.py

```diff
@@ -0,0 +1,43 @@
+import time
+
+import requests
+import os
+import argparse
+
+
+DATADOG_AGENT_PIPELINE_ID = 22
+BASE_URL = os.environ['CI_API_V4_URL']
+CI_TOKEN = os.environ['CI_JOB_TOKEN']
+
+
+def trigger_pipeline():
+    data = {
+        "token": CI_TOKEN,
+        "ref": "master",
+        "variables": {
+            "RELEASE_VERSION_6": "nightly",
+            "RELEASE_VERSION_7": "nightly-a7",
+            "DEB_RPM_BUCKET_BRANCH": "none",
+            "INTEGRATIONS_CORE_VERSION": os.environ['CI_COMMIT_REF_NAME'],
+        }
+    }
```

Comment:
I'd add a note for later: once the option to opt out of kitchen tests in the datadog-agent pipeline is available, we should use it here (because we shouldn't need kitchen tests to verify that the builds work).